apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -138,6 +153,10 @@ class GameLabVisualizationColumn extends React.Component {
             tabIndex="1"
             onMouseUp={this.pickerMouseUp}
             onMouseMove={this.pickerMouseMove}
+            onTouchMove={this.pickerTouchMove}
+            onTouchEnd={this.pickerTouchEnd}
+            onTouchCancel={this.pickerTouchEnd}
```

Comment:
Bummer that [iOS doesn't support Pointer Events](https://caniuse.com/#search=pointer%20events) yet.  I wonder if this is cleaner if we snap to using Pointer Events, with this polyfill for Safari?  https://github.com/jquery/PEP